Atomic/syscontainers.py

```diff
@@ -872,11 +866,14 @@ def _get_system_checkout_path(self):
             return ATOMIC_VAR
 
     def get_ostree_repo_location(self):
+        location = os.environ.get("ATOMIC_OSTREE_REPO")
+        if location is not None:
+            return location
+
         if self.user:
-            return "%s/.containers/repo" % HOME
+            return  "%s/.containers/repo" % HOME
         else:
```

Comment:
Nit.  The else isn't necessary.  You might want to consider dropping it and just doing the bare return here.